crates/ruff_linter/src/rules/ruff/rules/missing_fstring_syntax.rs

```diff
@@ -88,8 +88,10 @@ fn should_be_fstring(
         return false;
     }
 
+    let fstring_expr = format!("f{}", locator.slice(literal.range()));
```

Comment:
nit: we probably don't require the call to `range` because `locator.slice` expects a `Ranged` type

```suggestion
    let fstring_expr = format!("f{}", locator.slice(literal)));
```